hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/marker/DirectWriteMarkers.java

```diff
@@ -158,7 +162,10 @@ private String translateMarkerToDataPath(String markerPath) {
     Set<String> markerFiles = new HashSet<>();
     if (doesMarkerDirExist()) {
       FSUtils.processFiles(fs, markerDirPath.toString(), fileStatus -> {
-        markerFiles.add(stripMarkerFolderPrefix(fileStatus.getPath().toString()));
+        String filePathStr = fileStatus.getPath().toString();
+        if (!filePathStr.contains(MARKER_TYPE_FILENAME)) {
```

Comment:
Since we agreed that we don't need marker type file with Direct, do we still need this check ?

Reply:
This is no longer needed.  Removed.